--- a/fstab      2020-02-01 18:11:02.623889078 +0100
+++ b/fstab  2020-02-15 17:52:09.396489916 +0100
@@ -7,5 +7,5 @@
 tmpfs                /var/volatile        tmpfs      defaults              0  0
 
 # uncomment this if your device has a SD/MMC/Transflash slot
-#/dev/mmcblk0p1       /media/card          auto       defaults,sync,noauto  0  0
+/dev/mmcblk0p1       /boot          auto       defaults,sync,noauto  0  0
 
